### PR TITLE
Fix tower timestamp conversion

### PR DESCRIPTION
**Pull request by fevac**
## Description

- Error logging when converting tower timestamps was misplaced. 

### Changed

- Tower timestamp conversion logging and pytest

### How to prepare for test
- [x] ssh to hasta (depending on type of change)
- [x] activate stage: `us`
- [x] request trailblazer-stage on hasta: `paxa`
- [x] install on stage:
    ```Shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_trailblazer -t trailblazer -b fix_tower_timestamp_conversion -a
    ```

### How to test
- [x] `trailblazer scan` works

## Review
- [x] tests executed by EFC
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Review by islean — APPROVED** [review]
Looks like a quick fix 🔥

File: trailblazer/utils/datetime.py [hunk 1]
```diff
@@ -32,8 +30,9 @@ def get_datetime_from_timestamp(timestamp: str, datetime_formats: List[str]) ->
         try:
             return datetime.strptime(timestamp, datetime_format)
         except ValueError as error:
-            LOG.error(f"Error converting timestamp: {error}")
+            error_message = f"Error converting timestamp: {error}"
             continue
+    LOG.error(error_message)
```
[the remark this answers is not in the document]
Comment (fevac) [reply trailblazer/utils/datetime.py:35]: yes, I was thinking the same. Do we want to raise an error here @henrikstranneheim ?

[the remark this answers is not in the document]
Comment (fevac) [reply trailblazer/utils/datetime.py:35]: well I'll keep the current behaviour for now. We can change int the future if we want to

**Comment by fevac** [comment]
- Deployed in stage
image


- Deployed in prod

image

**Pull request merged**